### DISCUSSION TITLE
我想玩东方夜雀食堂和浮岛物语

### DISCUSSION THREAD

**Discussion by mmmgpx** [Gamuxorg/bbs#439] [投递和请求游戏]
我想玩东方夜雀食堂和浮岛物语
不要用红酒和质子
我的系统是linux mint

**Comment by Parollel** [comment]
这俩游戏官方都没提供Linux原生版本，浮岛物语连MacOS都没有。
除非可以逆向出代码来或者是无平台依赖的游戏引擎开发的，不然肯定要用Wine。

**Comment by hyh19962008** [comment]
浮岛物语已添加，东方系列好像都不支持Linux。浮岛物语最早的时候有过Linux版本，不过作者没有时间精力维护，后来不更新了，再后来就下架了。
参考：
https://www.gamingonlinux.com/2019/10/hopfrog-is-removing-linux-support-from-forager-and-macos-is-not-coming-now-either/page=2

**Comment by mmmgpx** [comment]
夜雀食堂可以上架用红酒或者质子吗？真的很想玩

获取Outlook for Android<https://aka.ms/AAb9ysg>
________________________________

**Reply by hyh19962008** [reply]
protondb上显示可以完美运行，另外还可以考虑switch模拟器的选项
